scripts/pipeline/orthology_benchmark.py

```diff
@@ -17,18 +17,134 @@
 
 Typical usage example::
 
-    $ python orthology_benchmark.py --mlss_conf /path/to/mlss_conf.xml \
-        --species_set default
+    $ python orthology_benchmark.py --mlss_conf /path/to/mlss_conf.xml --species_set name \
+    --host mysql-ens-compara-prod-X --port XXXX --user ensro --out_dir /path/to/out/dir
 
 """
 
 import argparse
+import os
+import subprocess
+from typing import Dict, List
+
+import MySQLdb
+import numpy
 
 from ensembl.compara.config import get_species_set_by_name
 
 
-def dump_genomes():
-    """Docstring"""
+def dump_genomes(species_list: List[str], species_set_name: str, host: str, port: int,
+                 user: str, out_dir: str) -> None:
+    """Dumps canonical peptides of protein-coding genes for a specified list of species
+    from the latest available core databases to FASTA files using `dump_gene_set_from_core.pl`.
```

Comment:
The advised format in docstrings is a short description in the first line, and then a larger description (if required). Something like:
```
    """Dumps canonical peptides of the given species' protein-coding genes.

    The dumping of each species is done for the latest available core databases to FASTA files
    using `dump_gene_set_from_core.pl`.
```